static/src/stylesheets/module/content/_media.global.scss

```diff
@@ -15,8 +15,28 @@
         }
     }
 
+    .video__fader {
+        position: fixed;
+        z-index: 9999;
+        top: 0;
+        bottom: 0;
+        left: 0;
+        right: 0;
+        background-color: rgba(0, 0, 0, .8);
+        opacity: 0;
+        transition: all 1s ease-in-out;
+        visibility: hidden;
+        
+        &.is-faded {
+            visibility: visible;
+            opacity: 1;
+            transition: all 1s ease-in-out;
+        }
+    }
+
     .player {
         margin-bottom: $gs-baseline/2;
+        z-index: 99999;
```

Comment:
Does this not cause problems when you open an image overlay with a video on the page? Feels like the player would be on top when it shouldn't be? Should you just apply a higher z-index when you know the fade is there? Also should you be checking whether a modal window is open or does it only appear when playing?